app/controllers/registered_applications_controller.rb

```diff
@@ -0,0 +1,29 @@
+class RegisteredApplicationsController < ApplicationController
+
+  def index
+    @registered_applications = RegisteredApplication.all
+  end
```

Comment:
Line 4 - How can this be modified so a user will only see their own registered applications?